package.json

```diff
@@ -638,6 +669,178 @@
                     "default": false
                   }
                 }
+              },
+              "pipeTransport": {
+                "type": "object",
+                "description": "When present, this tells the debugger to connect to a remote computer using another executable as a pipe that will relay standard input/output between VS Code and the .NET Core debugger backend executable (clrdbg).",
+                "default": {
+                  "pipeCwd": "${workspaceRoot}",
+                  "pipeProgram": "enter the fully qualified path for the pipe program name, for example 'c:\\tools\\plink.exe'",
+                  "pipeArgs": [],
+                  "debuggerPath" : "enter the path for the debugger, for example ~/clrdbg/clrdbg"
+                },
+                "properties": {
+                  "pipeCwd": {
+                    "type": "string",
+                    "description": "The fully qualified path to the working directory for the pipe program.",
+                    "default": "${workspaceRoot}"
+                  },
+                  "pipeProgram": {
+                    "type": "string",
+                    "description": "The fully qualified pipe command to execute.",
+                    "default": "enter the fully qualified path for the pipe program name, for example 'c:\\tools\\plink.exe'"
+                  },
+                  "pipeArgs": {
+                    "type": "array",
+                    "description": "Command line arguments passed to the pipe program.",
+                    "items": {
+                      "type": "string"
+                    },
+                    "default": []
+                  },
+                  "debuggerPath" : {
+                    "type" : "string",
+                    "description" : "The full path to the debugger",
+                    "default" : "~/clrdbg/clrdbg"
+                  },
+                  "pipeEnv": {
+                    "type": "object",
+                    "additionalProperties": {
+                      "type": "string"
+                    },
+                    "description": "Environment variables passed to the pipe program.",
+                    "default": {}
+                  },
+                  "windows": {
+                    "type": "object",
+                    "description": "Windows-specific pipe launch configuration options",
+                    "default": {
+                      "pipeCwd": "${workspaceRoot}",
+                      "pipeProgram": "enter the fully qualified path for the pipe program name, for example 'c:\\tools\\plink.exe'",
+                      "pipeArgs": [],
+                      "debuggerPath" : "enter the path for the debugger, for example ~/clrdbg/clrdbg"
```

Comment:
Thinking about this some more, there is no reason why 'debuggerPath' would be dependent on the host OS. So I would suggest removing it as a property from within the 'windows', 'osx', and 'linux' elements.